ddtrace/opentracer/tracer_test.go

```diff
@@ -23,3 +27,21 @@ func TestStart(t *testing.T) {
 	assert.True(ok)
 	assert.Equal(ott.Tracer, dd)
 }
+
+func TestSpanWithContext(t *testing.T) {
+	assert := assert.New(t)
+	ot := New()
+	_, ok := internal.GetGlobalTracer().(ddtrace.Tracer)
+	assert.True(ok)
+	ott, ok := ot.(*opentracer)
+	assert.True(ok)
```

Comment:
These assertions are already covered in other tests. We can reduce it down to:
```suggestion
	ot := New().(*opentracer)
```